### DISCUSSION TITLE
Wordwrap Settings

### DISCUSSION THREAD

**Discussion by dehnes** [MeanderingProgrammer/render-markdown.nvim#265] [Q&A] [answered]
Hi, can the plugin help me setting a word wrap only for md files? How could I achive this? Thanks in advance!

**Comment by MeanderingProgrammer** [comment, marked as answer]
Yes, you can use this plugin to modify whatever window options you want. This would work similar to adding an ftplugin `:h ftplugin`.

Add the following to your config:

```lua
require('render-markdown').setup({
    win_options = {
        wrap = { default = false, rendered = false },
    },
})
```

**Reply by dehnes** [reply]
Perfect, thanks a lot!!
